cmd/hercules/plugin.template

```diff
@@ -81,11 +83,17 @@ func ({{.varname}} *{{.name}}) Description() string {
 
 // Configure applies the parameters specified in the command line. Map keys correspond to "Name".
 func ({{.varname}} *{{.name}}) Configure(facts map[string]interface{}) error {
+  if l, exists := facts[hercules.ConfigLogger].(hercules.Logger); exists {
+		{{.varname}}.l = l
+	}
   return nil
 }
 
 // Initialize resets the internal temporary data structures and prepares the object for Consume().
 func ({{.varname}} *{{.name}}) Initialize(repository *git.Repository) error {
+  if {{.varname}}.l == nil {
```

Comment:
We should always overwrite anything existing in `Initialize()`, that's the method's philosophy